### PR TITLE
Feature preview page and navigation cleanup

### PR DESCRIPTION
**Pull request by aapeliv**
* Adds a "feature preview" page for feature flags/experimental stuff
* Cleans up navigation stuff, so that all menus are defined in a consistent way

I want to use the feature previews:

* Try out the SMS verification stuff before it's rolled out to everyone
* Settings page for new notification system before it replaces the old email-only stuff

**Web frontend checklist**
- [ ] Formatted my code with `yarn format && yarn lint --fix`
- [ ] There are no warnings from `yarn lint`
- [ ] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [ ] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

**Comment by lucaslcode** [comment]
Can this wait until next is merged?

**Comment by aapeliv** [comment]
> Can this wait until next is merged?

Sure

File: app/web/.prettierignore [hunk 1]
```diff
@@ -2,3 +2,9 @@ proto/
 node_modules
 **/locales/*.json
 .next/
+coverage/
+junit.xml
+dl-protos.sh
+*.Dockerfile
+*.log
+yarn.lock
```
Comment (darrenvong) [inline app/web/.prettierignore:10]:
```suggestion
```
We should keep the lock file committed like before!

File: app/web/features/auth/FeaturePreview.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import Section from "features/auth/section/Section";
+import { useTranslation } from "react-i18next";
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  section: {
+    margin: theme.spacing(4, 0),
+    "&:first-of-type": {
+      marginTop: theme.spacing(2),
+    },
```
Comment (darrenvong) [inline app/web/features/auth/FeaturePreview.tsx]:
```suggestion
```
Seems like this is redundant to me

**Review by darrenvong — CHANGES_REQUESTED** [review]
Only some small things, rest looks good to me! Nice to see this start adding to the translation JSON too 🚀

File: app/web/.prettierignore (hunk 1 above)
Comment (aapeliv) [reply app/web/.prettierignore:10]: These are all committed, but this just stops prettier from looking at them. It seems when it doesn't find a formatter like for this extension, it'll return an error code

File: app/web/features/auth/FeaturePreview.tsx (hunk 2 above)
Comment (aapeliv) [reply app/web/features/auth/FeaturePreview.tsx]: Fixed

File: app/web/.prettierignore (hunk 1 above)
Comment (darrenvong) [reply app/web/.prettierignore:10]: Ahh yeah my bad! Misread that file as `gitignore`

**Review by darrenvong — APPROVED** [review]

**Pull request merged**